docs/pre_build_command_example.md

```diff
@@ -1,4 +1,4 @@
-Example `build.sh` that uses a secondary `Dockerfile.build` to generate artifacts the main `Dockerfile`
+Example `build_binary.sh` that uses a secondary `Dockerfile.build` to generate artifacts the main `Dockerfile`
```

Comment:
building directly on samson also works and would avoid this whole docker mess, but it's more brittle ...
I named it `build_binary.sh` so existing stages will opt-out of binary builder autoatically
the call to build_binary.sh should be in a command ... I'll add that ...